projects/com.oracle.truffle.llvm.runtime/src/com/oracle/truffle/llvm/runtime/floating/LLVM80BitFloat.java

```diff
@@ -626,4 +642,108 @@ public static LLVM80BitFloat fromString(String stringValue) {
         return fromBytesBigEndian(DatatypeConverter.parseHexBinary(stringValue));
     }
 
+    protected abstract static class LLVM80BitFloatNativeCallNode extends LLVMNode {
+        private final String name;
+
+        @Child private Node nativeExecute = Message.createExecute(2).createNode();
```

Comment:
I would not have this as a field but rather create an execute node using `@Cached`

Reply:
I'm not sure... There is only a single specialization, and it's always used, so I think it doesn't matter here. Unless I'm missing something?